examples/simple_example/README.md

```diff
@@ -0,0 +1,27 @@
+[^]: (autogen_docs_start)
```

Comment:
This should include a description of what the example module demonstrates.

Reply:
Addressed in 756ab6f.